### ISSUE TITLE
Gulp-testcafe fails with httpAuth

### ISSUE THREAD

**Issue by ghiscoding** [DevExpress/testcafe#1308]
### Are you requesting a feature or reporting a bug?
Reporting a bug

### What is the current behavior?
Running with `Gulp-Testcafe` with `httpAuth()` function, fails and throws an error
```bash
Cannot prepare tests due to an error.
TypeError: fixture(...).page(...).httpAuth is not a function
```
However there is not error while using `testcafe chrome test/test.js` in the shell

### What is the expected behavior?
To run with Gulp-Testcafe

### How would you reproduce the current behavior (if this is a bug)?
install `Gulp-Testcafe` and use the `httpAuth()` function will produce the error when running gulp

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

Gulp
```js
const gulp = require('gulp');
const testcafe = require('gulp-testcafe');

gulp.task('e2e', () => {
    return gulp.src('test/test.js')
        .pipe(testcafe({ browsers: ['chrome', 'ie'] }));
});
```
Spec file
```js
import { Selector } from 'testcafe';

fixture `Getting Started`
    .page `http://localhost/myproject`
    .httpAuth({
        username: 'myUsername',
        password: 'myPassword'
    });

test('Country List', async t => {
    await t.click('#adminCountryList');

    const location = await t.eval(() => window.location);
    await t.expect(location.pathname).eql('/myproject/country/list');
    const pageHeader = await Selector('h2');
   
    let headerText = await pageHeader.innerText;
    await t.expect(headerText).eql('Country List');
```

### Specify your

* operating system: Windows 10
* testcafe version: "testcafe": "^0.13.0", "gulp-testcafe": "^0.9.0",
* node.js version: 6.9.5

**Comment by inikulin** [comment]
@ghiscoding This is caused by the fact that gulp-testcafe uses outdated version of testcafe. Thank you for the report!

**Comment by inikulin** [comment]
I've just published `gulp-testcafe@9.0.1`. It's now always uses latest testcafe version.

**Comment by ghiscoding** [comment]
Thanks that worked :)
